acl_loader/main.py

```diff
@@ -676,7 +676,14 @@ def show_rule(self, table_name, rule_id):
         header = ("Table", "Rule", "Priority", "Action", "Match")
 
         def pop_priority(val):
-            priority  = val.pop("PRIORITY")
+            priority = 0
+            for key in dict(val):
```

Comment:
what if we will do:
```python
val = {k.upper(): v for k, v in val}
```
And fix it for any ACL rule attribute?

Reply:
This may not be required to set for all the ruleprops of ACL rule.